### ISSUE TITLE
Focus in toolbar does not cycle

### ISSUE THREAD

**Issue by CKFinderBOT** [ckfinder/ckfinder#239]
Having "Upload" button focused it should be possible to jump to the last focusable item by pressing `left` or `up` key.

**Comment by CKFinderBOT** [comment]
Fixed. The updated code will be publicly available in the upcoming release.
